engine/servermaster/config.go

```diff
@@ -39,18 +40,21 @@ const (
 	defaultDiscoverTicker    = 3 * time.Second
 	defaultMetricInterval    = 15 * time.Second
 	defaultMasterAddr        = "127.0.0.1:10240"
+	defaultEtcdEndpoint      = "127.0.0.1:2379"
 
 	// DefaultBusinessMetaID is the ID for default business metastore
 	DefaultBusinessMetaID        = "_default"
 	defaultBusinessMetaEndpoints = "127.0.0.1:3336"
 	defaultBusinessMetaUser      = "root"
 	defaultBusinessMetaPassword  = ""
+	defaultBusinessMetaSchema    = "test"
```

Comment:
I find there exists same table in business meta and framework meta, it is better to use different default schema, such as `test_business` and `test_framework`